content/webapp/services/prismic/types/index.ts

```diff
@@ -222,6 +222,12 @@ export function isFilledLinkToDocumentWithData<T, L, D extends DataInterface>(
   return 'id' in field && field.isBroken === false && 'data' in field;
 }
 
+export function isFilledLinkToDocument<T, L>(
```

Comment:
I suspect a bunch of these helper functions are provided by newer versions of the Prismic client libs, when we can update to them.